lib/git_reflow/git_server/git_hub/pull_request.rb

```diff
@@ -78,6 +78,45 @@ def approved?
           end
         end
 
+        def merge_feature_branch(feature_branch_name, options = {})
```

Comment:
@codenamev I am not exactly sure what the format is to get the: `commit_title, commit_message, sha, commit_number` from `pull_request.rb`. I tried looking in the code for examples, but couldn't find any. Do you just access these attributes from `self.source_object[:title]`? I was just wondering if you could provide an example of this? 
Thanks so much!